source/js/buyers-guide/components/creep-vote/creep-vote.jsx

```diff
@@ -159,6 +160,18 @@ export default class CreepVote extends React.Component {
    * @returns {jsx} What users see when they have voted on this product.
    */
   renderDidVote(){
+    let numGroups = 5;
```

Comment:
Pomax landed a PR that has the creepiness groups as a constant- can we leverage that so we're not replicating it here?

Reply:
@alanmoo can you point me where that variable is? I can't find it in `creep-vote.jsx`

Reply:
https://github.com/mozilla/foundation.mozilla.org/pull/1967/files (creepiness-labels.js)